test_runner/fixtures/neon_fixtures.py

```diff
@@ -469,6 +469,7 @@ def init_start(
         default_remote_storage_if_missing: bool = True,
         initial_tenant_shard_count: int | None = None,
         initial_tenant_shard_stripe_size: int | None = None,
+        timeout_in_seconds: int | None = None,
```

Comment:
@bayandin I could make it here `default 30s`:

wdyt?

```suggestion
        timeout_in_seconds: int | None = 30,
```

Reply:
Another observation is that all of those 60 failures happened for debug builds, so we should set a larger timeout for that `build_type`.
I'd like to get John's opinion on that first